crypto/cryptonight_aesni.h

```diff
@@ -300,7 +300,7 @@ static __forceinline __m128i int_sqrt33_1_double_precision(const uint64_t n0)
 
 	uint64_t x2 = (s - (1022ULL << 32)) * (r - s - (1022ULL << 32) + 1);
 	//if (x2 < n0) ++r;
-	_addcarry_u64(_subborrow_u64(0, x2, n0, &x2), r, 0, &r);
+	_addcarry_u64(_subborrow_u64(0, x2, n0, (unsigned long long int*)&x2), r, 0, (unsigned long long int*)&r);
```

Comment:
It's better to write uint64_t here just like in the rest of the function.

Reply:
Or not... I guess it requires unsigned long long int here to compile successfully on gcc.

Reply:
Yes `uint64_t*` is not on all system `unsigned long long int*`